### PR TITLE
Allow configuration of the Kubernets stability check duration

### PR DESCRIPTION
**Pull request by ellimist**
Allow configuration of the `STABILITY_CHECK_DURATION` via env variables

/cc @zendesk/samson

### Tasks
 - [ ] :+1: from team

### References
 - Jira link:

### Risks
- Level: Low

### CODE REVIEW

**Comment by ellimist** [comment]
Tests are all kind of twisted though. @grosser do you have any ideas?

File: plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb [hunk 1]
```diff
@@ -665,6 +665,17 @@ def worker_is_unstable
       assert executor.send(:stable?)
     end
 
+    it "overrides the stability time check via env var" do
```
Comment (grosser) [inline plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb]: best split that into 2 tests

Comment (grosser) [reply plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb]: also test KUBERNETES_STABILITY_CHECK_TICK

File: plugins/kubernetes/app/models/kubernetes/deploy_executor.rb [hunk 2]
```diff
@@ -10,8 +10,8 @@ class DeployExecutor
     end
     WAIT_FOR_LIVE = Integer(ENV.fetch('KUBERNETES_WAIT_FOR_LIVE', '600'))
     WAIT_FOR_PREREQUISITES = Integer(ENV.fetch('KUBERNETES_WAIT_FOR_PREREQUISITES', WAIT_FOR_LIVE))
-    STABILITY_CHECK_DURATION = 1.minute
-    TICK = 2.seconds
+    STABILITY_CHECK_DURATION = Integer(ENV.fetch('KUBERNETES_STABILITY_CHECK_DURATION', 1.minute))
```
Comment (grosser) [inline plugins/kubernetes/app/models/kubernetes/deploy_executor.rb]: `1.minute.to_i` might work

Comment (grosser) [reply plugins/kubernetes/app/models/kubernetes/deploy_executor.rb]: nvm

**Comment by ellimist** [comment]
- Removed the tests 😢  
- Updated the documentation ✅

**Review by miketineo — APPROVED** [review]

**Pull request merged**